app/renderer/lib/tabUtil.js

```diff
@@ -36,3 +36,8 @@ module.exports.hasBreakpoint = (breakpoint, arr) => {
   arr = Array.isArray(arr) ? arr : [arr]
   return arr.includes(breakpoint)
 }
+
+module.exports.hasTabAsRelatedTarget = (event) => {
+  const tabComponents = /tab(?=_|area|title|id)|closetab|icon/i
```

Comment:
this seems fragile to me, we should probably be checking for children with something like https://stackoverflow.com/questions/4697758/prevent-onmouseout-when-hovering-child-element-of-the-parent-absolute-div-withou